monai/transforms/utility/array.py

```diff
@@ -590,10 +595,12 @@ def __init__(
         self.additional_info = additional_info
         self._logger_name = "DataStats"
         _logger = logging.getLogger(self._logger_name)
-        _logger.setLevel(logging.INFO)
-        console = logging.StreamHandler(sys.stdout)  # always stdout
-        console.setLevel(logging.INFO)
-        _logger.addHandler(console)
+        if logging.root.getEffectiveLevel() > logging.INFO:
+            # if the root log level is higher than INFO, set a separate stream handler to record
+            _logger.setLevel(logging.INFO)
+            console = logging.StreamHandler(sys.stdout)
```

Comment:
This is the same code as the original but I wonder if hard-coding changing the log level of `_logger` to INFO and writing to stdout is desirable.

Reply:
Changed to always `setLevel(logging.INFO)` in the latest commit.
Checking the `root.level` is to avoid adding duplicated stdout handler when root level is already INFO.

Thanks.